usecases/schema/file_structure_migration.go

```diff
@@ -170,7 +146,12 @@ type fileMatcher struct {
 	classes             map[string][]*classShard
 }
 
-func newFileMatcher(schemaGetter schema.SchemaGetter, rootPath string) *fileMatcher {
+type schemaGetter interface {
+	CopyShardingState(class string) *sharding.State
+	GetSchemaSkipAuth() entschema.Schema
+}
```

Comment:
**suggestion:**  what do you think if we move that to manager and use it it from there because i see we have duplication  of that interface in multiple places like [here](https://github.com/weaviate/weaviate/blob/master/usecases/schema/manager.go#L67)

```go
type ClusterGetter interface {
	CopyShardingState(class string) *sharding.State
	GetSchemaSkipAuth() entschema.Schema
}

type SchemaGetter interface {
	ClusterGetter 
	...
```

Reply:
I will resolve this comment and merge this fix